dhall/src/Dhall/Parser/Expression.hs

```diff
@@ -840,9 +881,8 @@ local = do
 http :: Parser ImportType
 http = do
     url <- httpRaw
-    headers <- optional (try $ do
-        whitespace
-        _using
+    headers <- optional (do
+        try (whitespace *> _using)
```

Comment:
I think this would fail to parse something like

    http://example.com usingBla

Not that I would expect to see that in practice! ;)

It could be fixed though by including the subsequent `nonemptyWhitespace` in the `try`.